scss/misc/_keyboard_hint_box.scss

```diff
@@ -0,0 +1,93 @@
+.keyboard-hints-popover,
+.keyboard-hints-box,
+.keyboard-hint-cheatsheet-dialog {
+  .keybinding {
+    display: block;
+    margin-bottom: 7px;
+
+    .key {
+      display: inline-block;
+      padding: 3px 10px;
+      border-radius: 4px;
+      background: lightgrey;
+      box-shadow: 0px 3px rgba(0, 0, 0, 0.35);
+      border: 1px grey solid;
+
+      &:hover,
+      &.pressed {
+        box-shadow: 0px 1px rgba(0, 0, 0, 0.35);
+        transform: translatey(2px);
+      }
+    }
+  }
+  .action {
+    display: block;
+    background-color: rgba(0, 0, 0, 0.05);
+    border-radius: 4px;
+    padding: 4px 6px;
+    margin: 4px 0px;
```

Comment:
I normally try to group css into following order:
positioning & size {position, display, width, margin...}
[flex] {flex-*, justify-*}
styling {border*, background, color...}
font {font-*}

I think having this somehow consistent helps a lot with readability. But mainly wanted to write down my "system" for this.